app/src/main/res/values/strings.xml

```diff
@@ -21,11 +21,19 @@ Please report any problems you encounter via Github.</string>
     <!-- Slide 2 -->
     <string name="storage_permission_title">Storage Permission</string>
     <string name="storage_permission_desc">Syncthing needs to access your storage to do file synchronization.</string>
+    <string name="dialog_all_files_access_not_supported">Your device does not support all files access</string>
 
     <!-- Slide 3 -->
     <string name="location_permission_title">Location Permission (background)</string>
     <string name="location_permission_desc">Syncthing can be configured to synchronize on selected Wi-Fi networks only. To do that it needs to look up the SSID of the currently connected Wi-Fi, also when the app is closed. Therefore it requires permission to access the location in the background. This information is not used to look up your location. And the only stored information is the Wi-Fi you manually choose to add to the list - no other user data is stored.\n\n If you want to use this feature, press the button above to grant the required permissions to Syncthing. Otherwise you can skip this step.</string>
 
+    <!-- Slide 4 -->
+    <string name="api_level_30_title">Upgrade Information</string>
+    <string name="api_level_30_desc">With the latest update syncthing-android is targeting API level 30, which changes the behaviour of Android\'s filesystem. Good news: On Android 11 and newer it is now able to write to an external SD card. Bad news: It may be slower and some paths will be hidden. Due to these hidden paths, the database needs to be reset to prevent data loss. This will cause all your data to be hashed again, but it doesn\'t need to be transferred from remote devices. It also may create conflicts and re-create recently deleted files if you aren\'t fully in sync right now. Depending on how much data you have, this process can take quite a long time.\n\n Please press the button below to reset the database and start Syncthing.</string>
```

Comment:
Worth noting that it will be an intensive process that will make the phone hot/drain battery?
Wouldn't want to press that button while on my last 3% while waiting for uber